plugins/module_utils/elbv2.py

```diff
@@ -719,11 +719,46 @@ def _compare_condition(self, current_conditions, condition):
         condition_found = False
 
         for current_condition in current_conditions:
-            if current_condition.get('SourceIpConfig'):
```

Comment:
Can you raise an error if the boto3 version is 1.16.57 <. I think this would save a lot of painful troubleshooting for the users.

For instance:  https://github.com/ansible-collections/community.aws/blob/main/plugins/modules/ec2_launch_template.py#L689-L690

Reply:
Thanks for the reference. I've added a boto3 version check when the ElasticLoadBalancerV2 class is initialised. Let me know if it'd be better placed elsewhere.